packages/ux-capture/README.md

```diff
@@ -144,6 +144,8 @@ Call UXCapture.mark in the HTML markup for each ‘mark’ name passed into
 ...
 ```
 
+**NOTE:** It is OK to fire marks before view is configured. This still works because UXCapture remembers all the recorded marks irrespective of whether it was first declared or not because ExpectedMark instances are created and cached when UXCapture.mark() is called. When UXCapture.startView() is executed and zone-mark dependencies are established, mark's callbacks are immediately fired if mark was already cached and marked as marked.
```

Comment:
```suggestion
**NOTE:** It is OK to fire marks before a view is configured. This still works because UXCapture remembers all the recorded marks irrespective of whether it was first declared or not because ExpectedMark instances are created and cached when `UXCapture.mark()` is called. When `UXCapture.startView()` is executed and zone-mark dependencies are established, mark's callbacks are immediately fired if mark was already cached and marked as marked.
```
I don't fully follow here...what is `ExpectedMark` (I don't see it referenced elsewhere in the readme)? And the `mark's callbacks are immediately fired if mark was already cached and marked as marked` part.

Is this an accurate summary? This may be all we need to say:
```
**NOTE:** UXCapture will record and remember `UXCapture.mark()` calls even before the page's UXCapture configuration has executed. This means UXCapture can be configured at the bottom of the body tag.
```

Reply:
Or, does this mean that UXCapture configuration doesn't need to be inlined at all?

Reply:
ExpectedMark is basically a map of all the expected marks on a page that have to be recorded before a zone is considered complete https://github.com/ux-capture/ux-capture/blob/main/packages/ux-capture/src/ExpectedMark.js#L25-L32 

Each ExpectedMark on the page has a marked boolean property that indicates if the mark has already been recorded. 
If the mark has already been recorded, we need to call the listener directly to ensure that the zones which depend on these marks are still marked as complete and not dropped https://github.com/ux-capture/ux-capture/blob/main/packages/ux-capture/src/ExpectedMark.js#L92-L96

Your revised summary seems accurate to me but i'll defer to Sergey for final word on this.